openfisca_core/legislation.xsd

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+    <xs:element name="NODE" type="nodeType"/>
+
+    <xs:group name="piecewiseConstant">
```

Comment:
What is a `piecewiseConstant ` ? I've seen this term a lot in this PR, but I don't really get what it represents.